src/SFA.DAS.Commitments.Api/DependencyResolution/DefaultRegistry.cs

```diff
@@ -19,7 +19,6 @@
 using System.Web;
```

Comment:
using System.Runtime.InteropServices.ComTypes; above this line isn't used and can be deleted.

Reply:
Removed.